skimage/measure/_marching_cubes_lewiner.py

```diff
@@ -0,0 +1,204 @@
+import sys
+import base64
+
+import numpy as np
+
+if sys.version_info[0] == 3:
+    base64decode = base64.decodebytes
+else:
+    base64decode = base64.decodestring
+
+from . import _marching_cubes_lewiner_luts as mcluts
+from . import _marching_cubes_lewiner_cy
+
+
+
+def marching_cubes_lewiner(volume, level=None, spacing=(1., 1., 1.),
+                           gradient_direction='descent', step_size=1,
+                           allow_degenerate=True, use_classic=False):
+    """
+    Lewiner marching cubes algorithm to find surfaces in 3d volumetric data
+    
+    In contrast to ``marching_cubes()``, this algorithm resolves
+    ambiguities and guarantees topologically correct results.
```

Comment:
I'd actually be fine with an addition here stating something like "if you aren't sure which to use, use this one." It's that much better than the original algorithm.